packages/docs/celo-codebase/protocol/proof-of-stake/carbon-offsetting-fund.md

```diff
@@ -2,4 +2,4 @@
 
 The Carbon Offsetting Fund provides for making the infrastructure of the Celo platform carbon-neutral, by making a transfer every epoch to an organization that commits to using those assets off-chain for carbon offsetting projects.
 
-Through the on-chain governance process, Celo Gold holders can set an on-target carbon offset amount, and the address of a carbon offsetting partner to which to direct these transfers. The on-target amount is adjusted by the epoch [rewards mulitplier](epoch-rewards.md), as with all epoch rewards.
+Through the on-chain governance process, Celo Gold holders can set the fraction of the total desired epoch rewards, initially planned to be 0.5%, that is received by the carbon offset, and the address of a carbon offsetting partner to which to direct these transfers. The on-target amount is adjusted by the epoch [rewards mulitplier](epoch-rewards.md), as with all epoch rewards.
```

Comment:
missing "fund" after "that is received by the carbon offset"

Reply:
An opportunity to use my favorite GitHub feature!
```suggestion
Through the on-chain governance process, Celo Gold holders can set the fraction of the total desired epoch rewards, initially planned to be 0.5%, that is received by the carbon offset fund, and the address of a carbon offsetting partner to which to direct these transfers. The on-target amount is adjusted by the epoch [rewards mulitplier](epoch-rewards.md), as with all epoch rewards.
```

Reply:
Thanks for catching this! And pretty neat feature indeed.